docs/manage/task-config-policies.rst

```diff
@@ -0,0 +1,165 @@
+still maintaining policy-based control over resource usage.
+
++------------------------------------------------------------------+
+| Visit the Config Policies Reference                              |
++==================================================================+
+| :ref:`task-config-policies-reference`                            |
++------------------------------------------------------------------+
+
+**************
+ Key Features
+**************
+
+-  Set policies via WebUI or CLI
+-  Define limits for resource usage, environment settings, and more
+-  Apply policies at different levels (cluster-wide or workspace)
+-  Override capabilities for specific user roles or groups
+-  Set different priority limits for experiments and NTSC (notebooks, TensorBoards, shells, and
+   commands) tasks
+-  Allow priority overrides within policy constraints
+
+******************
+ Setting Policies
+******************
+
+Administrators can set Config Policies using either the WebUI or the CLI.
+
+WebUI
+=====
+
+#. Sign in to the Determined WebUI as a cluster administrator or workspace administrator.
+#. Navigate to **Config Policies**.
+#. Choose **Experiments** or **Tasks** to display the editable configuration file.
+#. Define the policies then click **Apply**.
+#. Confirm you want to apply the policies.
+
+CLI
+===
+
+Use the following commands to manage Task Config Policies via CLI:
+
+.. code:: bash
+
+   # List existing policies
+   det policy list
+
+   # Create a new policy
+   det policy create <policy_name> --config <policy_config_file.yaml>
+
+   # Update an existing policy
+   det policy update <policy_name> --config <updated_policy_config_file.yaml>
+
+   # Delete a policy
+   det policy delete <policy_name>
+
+*****************
+ Policy Examples
+*****************
+
+Here are some example policies:
+
+#. Limit resources.
+
+In the following example for an Agent resource manager (RM), the ``priority_limit`` is set to ``50``
+and ``max_slots`` is set to ``1``. This means a user cannot set a priority value lower than 50 and
+cannot set ``max_slots`` to greater than ``1``.
+
+.. note::
+
+   Tasks (NTSC - notebooks, Tensorboards, shells, and commands) and some resource managers (RMs)
+   have a priority. Priority levels are between 1 and 99. The default cluster priority is usually
+   42. The priority limit determines the order in which queued tasks run. In this example we are
+   using limits because for the agent resource manager, a lower number means a higher priority. For
+   a Kubernetes resource manager, a higher number means a higher priority.
+
+.. code:: bash
+
+   # Set priority limit to 50
+   constraints:
+      priority_limit: 50
+      resources:
+         max_slots: 1
+
+If your priority limits are above the defaults, for example if your notebook priority is set to 42
+and the limit is 50, then the request fails and the system displays a descriptive message.
```

Comment:
same as above; let's use different numbers here

Notebook priority set to 10 and limit set to 15 works. As long as the limit is 30 or less, I am ok.